js/app/src/lite/index.ts

```diff
@@ -159,4 +161,9 @@ demo.launch()
 		controlPageTitle: false,
 		appMode: true
 	});
+
+	exec_button.onclick = (): void => {
+		console.debug("exec_button.onclick");
+		controller.runPython(code_input.value);
```

Comment:
@aliabd Upon here, I would like to ask you, who created the online editor demo with the very first ver. of Gradio-lite, about this API design and the method names. Does this make sense, or are there something to improve?

Basically, currently, we can use the Gradio-lite's JS API as below.
```ts
// Initialization
const controller = create({
  // ...other options...
  pyCode: "<The initial code>",
})

// For subsequent Python code changes, use `controller.runPython()` as below.
if (whenToUpdate) {
  controller.runPython(code_input.value);
}
```

Reply:
Yes, such a tag-based one can also be implemented, and it may be a wrapper layer of the current `create()` function.
On the other hand, the feature introduced in this PR would be a different and specialized one for quick-reloading functionality which some specific types of applications needs, such as an online editor.

Reply:
You asked about API though, if this is internal then the API doesn't matter too much as long as it works.

Reply:
OK, so please merge it. I just wanted to ping to one of the main potential developers who use this feature for some opinions.

Reply:
I think it's fine to have a programmatic interface as well. The only thing I will say is that that should be snake case to match our other libraries and apis.

Reply:
Renamed it!